File: Examples/VisualBasic/WorkingWithCurrencies/GetCurrencyDigits.vb

Namespace WorkingWithCurrencies
    Public Class GetCurrencyDigits
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)

            ' ExStart:GetCurrencyDigits
            Dim project1 As Project = New Project(dataDir + "project.mpp")
            Console.WriteLine(project1.Get(Prj.CurrencyDigits))
            ' ExEnd:GetCurrencyDigits
        End Sub
    End Class
End Namespace

File: Examples/CSharp/WorkingWithCurrencies/GetCurrencyDigits.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithCurrencies
{
    using System;

    internal class GetCurrencyDigits
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());

            //ExStart:GetCurrencyDigits
            //ExFor: Prj.CurrencyDigits
            //ExSummary: Shows how to get the project currency digits.
            var project = new Project(dataDir + "project.mpp");
            Console.WriteLine(project.Get(Prj.CurrencyDigits));
            //ExEnd:GetCurrencyDigits
        }
    }
}
